onnxmltools/convert/sklearn/operator_converters/NaiveBayes.py

```diff
@@ -25,6 +25,7 @@ def convert_sklearn_naive_bayes(scope, operator, container):
     # input(or x): test set input
     # output(or y): test set output (There are two paths for producing output, one for
     #               string labels and the other one for int labels) 
+    # output_probability: test set class probabilties 
```

Comment:
Nit. What is test set? Is it relavent to your computational graph?

Reply:
Test set is what an onnx model takes as an input to give prediction. I am using the general naming convention of training / test set. The model was trained on the training set and now the instances we use as input is test set.

Reply:
No.. I mean, there is no test set for a computation graph. Can we feed training set into a computation graph? Yes, we can. Can we feed validation set into it? Yes, we can. Can we feed test set into a computation graph? Yes, we can. So, I still believe there is no need to put `test` into the doc string.